client/src/test/java/com/sinch/sdk/e2e/domains/voice/WebhooksEventsSteps.java

```diff
@@ -0,0 +1,193 @@
+package com.sinch.sdk.e2e.domains.voice;
+
+import com.sinch.sdk.core.TestHelpers;
+import com.sinch.sdk.domains.voice.WebHooksService;
+import com.sinch.sdk.domains.voice.models.CallReasonType;
+import com.sinch.sdk.domains.voice.models.CallResultType;
+import com.sinch.sdk.domains.voice.models.DestinationNumber;
+import com.sinch.sdk.domains.voice.models.Price;
+import com.sinch.sdk.domains.voice.models.webhooks.AnsweredCallEvent;
+import com.sinch.sdk.domains.voice.models.webhooks.DisconnectCallEvent;
+import com.sinch.sdk.domains.voice.models.webhooks.MenuInputType;
+import com.sinch.sdk.domains.voice.models.webhooks.MenuResult;
+import com.sinch.sdk.domains.voice.models.webhooks.MenuResultInputMethodType;
+import com.sinch.sdk.domains.voice.models.webhooks.PromptInputEvent;
+import com.sinch.sdk.domains.voice.models.webhooks.WebhooksEvent;
+import com.sinch.sdk.e2e.Config;
+import com.sinch.sdk.e2e.domains.WebhooksHelper;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import java.io.IOException;
+import java.net.URL;
+import java.time.Instant;
+import org.junit.jupiter.api.Assertions;
+
+public class WebhooksEventsSteps {
+
+  static final String WEBHOOKS_PATH_PREFIX = "/webhooks/voice";
+  static final String WEBHOOKS_URL = Config.VOICE_HOST_NAME + WEBHOOKS_PATH_PREFIX + "/";
```

Comment:
Nit: In my sense, it would be clearer to declare the `WEBHOOKS_URL` without the trailing `/` and to format the urls with (line 109):

```
new URL(WEBHOOKS_URL + "/pie-sequence"), service::unserializeWebhooksEvent);
```

Reply:
Yes it make sense
Done